examples/jsm/interactive/InteractiveGroup.js

```diff
@@ -53,19 +53,15 @@ class InteractiveGroup extends Group {
 		element.addEventListener( 'pointerdown', onPointerEvent );
 		element.addEventListener( 'pointerup', onPointerEvent );
 		element.addEventListener( 'pointermove', onPointerEvent );
-		element.addEventListener( 'mousedown', onPointerEvent );
-		element.addEventListener( 'mouseup', onPointerEvent );
-		element.addEventListener( 'mousemove', onPointerEvent );
 		element.addEventListener( 'click', onPointerEvent );
 
 		// WebXR Controller Events
-		// TODO: Dispatch pointerevents too
 
 		const events = {
-			'move': 'mousemove',
+			'move': 'pointermove',
 			'select': 'click',
```

Comment:
Maybe:
```js
'select': 'pointerdown',
```

Reply:
I think `pointerup` is more in line with the `select` event.

Reply:
Ok that was wrong, according to [this](https://developer.mozilla.org/en-US/docs/Web/API/XRSession/select_event) , you are right, it should be at the start.
However, I've run into a strange issue where the event gets fired twice, but only sometimes. I've narrowed it down to the select event triggering the function twice.